app-admin/stown/stown-1.2.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# shellcheck shell=bash disable=2034
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=(python3_{10..13})
+
+inherit distutils-r1 readme.gentoo-r1
+distutils_enable_tests unittest
```

Comment:
This should be just above phase definitions, not here. But doesn't this remove the need for `python_test`?

Reply:
See my comment above.